### DISCUSSION TITLE
My App Build stucks in "queue" status again

### DISCUSSION THREAD

**Discussion by phuongvubstar** [codemagic-ci-cd/codemagic-docs#2400] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I used Webhook to trigger to build. After receiving a hook, it stucks in the queue even though I click build manually

### Expected results

I hope the build will successfully. Please help me check if any issues occurs

### Actual results

Stuck in the queue status for a long time

**Comment by AndriesJacobus** [comment]
Same here...
Been waiting over an hour now

**Comment by dtrdic** [comment, marked as answer]
Hello there, the issue has been resolved, and the builds will start soon. Some builds might still be queued for a few minutes due to a high volume of builds starting simultaneously but they will all be processed. Sorry for the inconvenience and thank you for your patience
